pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -1394,6 +1534,11 @@ func (a *ApplicationBackupController) deleteBackup(backup *stork_api.Application
 		}
 		return true, err
 	}
+	// TODO: for nfs type, we need to invoke job based deletion.
+	// For now, skipping it.
```

Comment:
NFS vol type backup is also kdmp, can someone add more context as to what this comment means? Below code should take care of deleting nfs backups also 
```
canDelete, err := driver.DeleteBackup(backup)
		if err != nil {
			log.ApplicationBackupLog(backup).Errorf("%v", err)
			return canDelete, err
		}
```

Reply:
This for the deletion of resource files, if some one take back directly from stork. Currently it is not supported for NFS, We need invoke resource deletion job over here. Resolving for now. Will take it up later